### ISSUE TITLE
microsoft.logic/workflows missing "identity" property

### ISSUE THREAD

**Issue by alex-frankel** [Azure/azure-rest-api-specs#11986]
Originally reported here:
https://github.com/Azure/bicep/issues/949

Appears to be the case for all apiVersions, but a PUT with the identity property will work as expected. Here's their template sample from docs:
https://docs.microsoft.com/en-us/azure/templates/microsoft.logic/2019-05-01/workflows

**Comment by alex-frankel** [comment]
Hi @raych1 - can you confirm that this has been successfully routed to the logic apps team? It looks like the @ mention did not work. No other participants have been added to the issue.

**Comment by alex-frankel** [comment]
Hi logic apps team – can you all take a look at this issue?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by raych1** [comment]
@rarayudu, @pankajsn, @tonytang-microsoft-com, I found you're either code owners of `microsoft.logic` RP or swagger author. Can you have a look at this issue or re-assign properly. Thanks.

**Comment by alex-frankel** [comment]
Ping 😊 Can anyone take a look at this?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Alex Frankel<mailto:alfran@microsoft.com>
Sent: Tuesday, December 22, 2020 7:08 AM
To: Azure/azure-rest-api-specs<mailto:reply@reply.github.com>; Azure/azure-rest-api-specs<mailto:azure-rest-api-specs@noreply.github.com>; Jon Fancey<mailto:Jon.Fancey@microsoft.com>; Nitasha Verma<mailto:nitashav@microsoft.com>; Tony Tang<mailto:tonytang@microsoft.com>; Rama Krishna Rayudu<mailto:rarayudu@microsoft.com>; Rohitha Hewawasam<mailto:rohithah@microsoft.com>
Cc: Author<mailto:author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] microsoft.logic/workflows missing "identity" property (#11986)

Hi logic apps team – can you all take a look at this issue?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by alex-frankel** [comment]
Yes. The swagger for the logic needs to be updated to include the identity details. We have a backlogs item for this.

Thanks, Rama.

From: Alex Frankel <alfran@microsoft.com>
Sent: Sunday, January 10, 2021 9:57 PM
To: Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>; Jon Fancey <Jon.Fancey@microsoft.com>; Nitasha Verma <nitashav@microsoft.com>; Tony Tang <tonytang@microsoft.com>; Rama Krishna Rayudu <rarayudu@microsoft.com>; Rohitha Hewawasam <rohithah@microsoft.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] microsoft.logic/workflows missing "identity" property (#11986)

Ping 😊 Can anyone take a look at this?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Alex Frankel<mailto:alfran@microsoft.com>
Sent: Tuesday, December 22, 2020 7:08 AM
To: Azure/azure-rest-api-specs<mailto:reply@reply.github.com>; Azure/azure-rest-api-specs<mailto:azure-rest-api-specs@noreply.github.com>; Jon Fancey<mailto:Jon.Fancey@microsoft.com>; Nitasha Verma<mailto:nitashav@microsoft.com>; Tony Tang<mailto:tonytang@microsoft.com>; Rama Krishna Rayudu<mailto:rarayudu@microsoft.com>; Rohitha Hewawasam<mailto:rohithah@microsoft.com>
Cc: Author<mailto:author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] microsoft.logic/workflows missing "identity" property (#11986)

Hi logic apps team – can you all take a look at this issue?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by alex-frankel** [comment]
Great – do you have an ETA for when it will be resolved?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Rama Krishna Rayudu<mailto:rarayudu@microsoft.com>
Sent: Sunday, January 10, 2021 10:09 PM
To: Alex Frankel<mailto:alfran@microsoft.com>; Azure/azure-rest-api-specs<mailto:reply@reply.github.com>; Azure/azure-rest-api-specs<mailto:azure-rest-api-specs@noreply.github.com>; Jon Fancey<mailto:Jon.Fancey@microsoft.com>; Nitasha Verma<mailto:nitashav@microsoft.com>; Tony Tang<mailto:tonytang@microsoft.com>; Rohitha Hewawasam<mailto:rohithah@microsoft.com>
Cc: Author<mailto:author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] microsoft.logic/workflows missing "identity" property (#11986)

Yes. The swagger for the logic needs to be updated to include the identity details. We have a backlogs item for this.

Thanks, Rama.

From: Alex Frankel <alfran@microsoft.com>
Sent: Sunday, January 10, 2021 9:57 PM
To: Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>; Jon Fancey <Jon.Fancey@microsoft.com>; Nitasha Verma <nitashav@microsoft.com>; Tony Tang <tonytang@microsoft.com>; Rama Krishna Rayudu <rarayudu@microsoft.com>; Rohitha Hewawasam <rohithah@microsoft.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] microsoft.logic/workflows missing "identity" property (#11986)

Ping 😊 Can anyone take a look at this?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Alex Frankel<mailto:alfran@microsoft.com>
Sent: Tuesday, December 22, 2020 7:08 AM
To: Azure/azure-rest-api-specs<mailto:reply@reply.github.com>; Azure/azure-rest-api-specs<mailto:azure-rest-api-specs@noreply.github.com>; Jon Fancey<mailto:Jon.Fancey@microsoft.com>; Nitasha Verma<mailto:nitashav@microsoft.com>; Tony Tang<mailto:tonytang@microsoft.com>; Rama Krishna Rayudu<mailto:rarayudu@microsoft.com>; Rohitha Hewawasam<mailto:rohithah@microsoft.com>
Cc: Author<mailto:author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] microsoft.logic/workflows missing "identity" property (#11986)

Hi logic apps team – can you all take a look at this issue?

Thanks,
Alex

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by miqm** [comment]
@alex-frankel do you know if the team has some ETA on fixing this?

**Comment by darren-johnson** [comment]
This is an issue for me too. Is there an update on this?

**Comment by alex-frankel** [comment]
Latest ETA from logic apps team is June

**Comment by Dave14393** [comment]
Is there any update on this?

**Comment by ecfan** [comment]
Ping @Azure/azure-logicapps-team

**Comment by alex-frankel** [comment]
Since this is taking so long to resolve, we are prioritizing https://github.com/Azure/bicep/issues/3000 as a workaround. Current ETA for the swagger fix is sometime between now and end of CY2021..

The bicep workaround we are hoping to get to in our 0.5 release.
